usaspending_api/broker/helpers/upsert_fabs_transactions.py

```diff
@@ -218,24 +215,3 @@ def insert_new_fabs(to_insert):
 
     return update_award_ids
 
-
-def upsert_fabs_transactions(ids_to_upsert, externally_updated_award_ids):
-    if ids_to_upsert or externally_updated_award_ids:
-        update_award_ids = copy(externally_updated_award_ids)
-
-        if ids_to_upsert:
-            with timer("inserting new FABS data", logger.info):
-                update_award_ids.extend(insert_all_new_fabs(ids_to_upsert))
-
-        if update_award_ids:
-            update_award_ids = tuple(set(update_award_ids))  # Convert to tuple and remove duplicates.
-            with timer("updating awards to reflect their latest associated transaction info", logger.info):
-                update_awards(update_award_ids)
-            with timer("updating award category variables", logger.info):
-                update_award_categories(update_award_ids)
-
-        with timer("updating C->D linkages", logger.info):
-            update_c_to_d_linkages("assistance")
-
-    else:
-        logger.info("Nothing to insert...")
```

Comment:
The biggest part of this PR is renaming this to upsert_transactions and moving it to a common location